Modules/BenchPress.Azure/Public/CustomAssertions/BeInLocation.ps1

```diff
@@ -31,12 +31,21 @@ function ShouldBeInLocation ($ActualValue, [string]$ExpectedValue, [switch] $Neg
     if ($Negate) { $succeeded = -not $succeeded }
     $failureMessage = "ConfirmResult is null or empty."
   }
+  elseif (-not $ActualValue.ResourceDetails.Location) {
+    [bool] $succeeded = $false
+
+    if ($Negate) { $succeeded = -not $succeeded }
+    $failureMessage = "Resource does not have a location property."
+  }
   else {
     # Location needs to be normalized to all lower cases and no spaces
+    # For both expected value and actual value
     $rawResourceLocation = $ActualValue.ResourceDetails.Location.ToLower()
```

Comment:
It turns out that there is no need to convert to lower case. By default, `-eq` uses case insensitive comparison. However, if you wanted to be explicit, you could use `-ieq`, which would work exactly the same way.

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-7.3#common-features

Reply:
good catch. will go with `-ieq`